lib/MBTA.rb

```diff
@@ -0,0 +1,52 @@
+class Trip
+  MBTA = {
+    "Red" => [
+      "Alewife", "Davis", "Porter", "Harvard", "Central", "Kendall/MIT", "Park Street", "South Station"
+    ],
+    "Green" => [
+      "Haymarket", "Government Center", "Park Street", "Boylston", "Arlington", "Copley"
+    ],
+    "Orange" => [
+      "North Station", "Haymarket", "Park Street", "State Street", "Downtown Crossing", "Chinatown", "Tufts Medical Center"
+    ]
+  }
+
+  def initialize(oriLine:, oriStop:, destLine:, destStop:)
```

Comment:
A few comments. Idomatic ruby would have:

- `snake_case` for variables, including keyword parameters
- full names, like `origin_line`. Characters are cheap. No need to be cryptic.